app/script/view_model/content/PreferencesDeviceDetailsViewModel.js

```diff
@@ -63,14 +63,17 @@ z.ViewModel.content.PreferencesDeviceDetailsViewModel = class PreferencesDeviceD
     this.activated_on = ko.observable(z.l10n.text(z.string.preferences_devices_activated_on));
   }
 
-  _update_activation_location(location) {
-    const location_content = `<span class='preferences-devices-activated-bold'>${location}</span>`;
-    this.activated_in(z.l10n.text(z.string.preferences_devices_activated_in, location_content));
+  _update_activation_location(location, template = z.string.preferences_devices_activated_in) {
+    const text = z.l10n.text(template, location);
+    const sanitizedText = z.util.StringUtil.splitAtPivotElement(text, location);
+    this.activated_in(sanitizedText);
   }
 
-  _update_activation_time(time) {
-    const time_content = `<span class='preferences-devices-activated-bold'>${z.util.format_timestamp(time)}</span>`;
-    this.activated_on(z.l10n.text(z.string.preferences_devices_activated_on, time_content));
+  _update_activation_time(time, template = z.string.preferences_devices_activated_on) {
+    const formattedTime = z.util.format_timestamp(time);
+    const text = z.l10n.text(template, formattedTime);
+    const sanitizedText = z.util.StringUtil.splitAtPivotElement(text, formattedTime);
+    this.activated_on(sanitizedText);
```

Comment:
Both `activated_on` and `activated_in` should be observableArrays, since that is what `z.util.StringUtil.splitAtPivotElement` seems to return